process-engine/bpmn-studio
# 🐛 Fix Updating Xml View

## What did you change?

This Pr fixes the bug that the xml view did not get updated.

Fixes: #567 

## How can others test the changes?

- Open up the BPMN-Studio
- Go to the Detail View of any diagram
- Have a look at the xml view
- Change the diagramm
- Have a look at the xml view
- Notice that it now updates again

## PR-Checklist

Please check the boxes in this list after submitting your PR:
